Cargo.toml

```diff
@@ -13,7 +13,7 @@ members = [
 resolver = "2"
 
 [workspace.package]
-version = "1.69.1"
+version = "1.69.7"
```

Comment:
Not sure why this isn't getting updated to reflect releases. 
If we go to a different cadence, we should maybe revisit this as a release date designation?

Reply:
I just haven't merged the release/1.69 branch yet because we've been kinda rapid firing releases. Leave this out and I'll merge it after the next release